boards/realtek/rts5912_evb/rts5912_evb_defconfig

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2024 Realtek Semiconductor Corporation, SIBG-SD7
+#
+
+# Enable RTS5912 image tool 
+CONFIG_REALTEK_RTS5912_IMG_HEADER=y
+CONFIG_REALTEK_RTS5912_OUTPUT_BIN=y
+CONFIG_REALTEK_RTS5912_IMG_HEADER_SPI_FREQ_MHZ_DIV8=y
+CONFIG_REALTEK_RTS5912_IMG_HEADER_SPI_READ_CMD_NORMAL=y
+
+# Serial Driver
+CONFIG_SERIAL=y
+
+# CONSOLE
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# LOGGING
+CONFIG_LOG=y
+CONFIG_LOG_MODE_MINIMAL=y
```

Comment:
logging should not be enabled by default

Reply:
Have removed it.